psi-ms.obo

```diff
@@ -22350,6 +22350,33 @@ name: mzTab-M
 def: "Expanded tabular result format for metabolomics experiments reporting quantitative summary data, MS features and identification evidence." [PMID:30688441, PMID:31525911, https://www.psidev.info/mztab]
 is_a: MS:1000914 ! tab delimited text format
 
+[Term]
+id: MS:1003390
+name: crosslinker cleavage
```

Comment:
I'm thinking that "crosslinker cleavage characteristics" is a better name for the intent of this concept?

Reply:
yes, that is perhaps better, 
@lutzfischer / @lars-kolbowski - any comments?

Reply:
I am not sure in how far that is more accurate. If you ask me to characterise a crosslinker cleavage behaviour I would think, more then just the observable stubs play in; like e.g. under what conditions it cleaves. 
Now you could claim the same for describing "crosslink cleavage". So guess what I want to say is - I don't see the benefit of one vs the other. 
In summary I am fine with either name.

Reply:
I'm also fine with either

Reply:
if @edeutsch is the only one with a preference and others don't mind then lets go with @edeutsch preference.

>  I don't see the benefit of one vs the other.

I feel like "crosslinker cleavage" sounds more like something that happened, and "crosslinker cleavage characteristics" sounds more like a description of what might happen, so that's why i thought the latter might be better?